templates/partials/usages.html

```diff
@@ -0,0 +1,34 @@
+<h2>What can you build?</h2>
+<dl class="features">
+  <div>
+    <dt>
+      <div>
+        {{ icon_macros::sprite_icon(name="gears") }}
+        GUI Applications
+      </div>
+    </dt>
+    <dd>Perfect integration of GTK, Libadwaita, Libgranite makes development super simple. Publish easily your apps on Flathub without bloated packages.</dd>
+    <div>
+      <a href="https://developer.gnome.org/documentation/tutorials/beginners/getting_started.html">GNOME getting started tutorial</a>
+      <a href="https://docs.elementary.io/develop/writing-apps/hello-world">Elementary OS app tutorial</a>
```

Comment:
It looks crowded already with two links that aren't inline. Maybe having one link to the tutorials section of the docs would look much neater and be future proof too?

Reply:
I am not sure about this. I think it helps beginners when they see "GNOME" or "Elementary OS" and can click on it. But I don't know.